packages/libs/sdk/src/api/api.ts

```diff
@@ -102,7 +110,34 @@ export class API {
     const cacheStructure = new InMemoryCache({
       possibleTypes: generatedPossibleTypes.possibleTypes,
       // TODO: Figure out type policies
-      typePolicies: {},
+      typePolicies: {
+        EVMSchemaType: {
+          // Always merge EVMSchemaType objects
+          // Added because of warning: "Cache data may be lost when replacing the ethereum field of a Query object."
+          merge: true,
+        },
+        NFT: {
+          keyFields: ['contractAddress', 'tokenId'],
+        },
+        Collection: {
+          keyFields: ['address'],
```

Comment:
Not sure if it happens, but do we have to worry about the same collection using the same contract address across chains being cached as the same thing?  but if you had "Foo" collection on Ethereum with "0x123" address and Polygon with "0x123" address, would it return the ethereum cached collection for a polygon query because the address is the same?